x-pack/plugins/threat_intelligence/public/modules/indicators/components/indicator_barchart_legend_action/styles.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CSSObject } from '@emotion/react';
+
+export const useStyles = () => {
+  const contextMenu: CSSObject = {
+    'margin-top': '-2px',
```

Comment:
this is quite weird that we need to do this hacks to align the components that are built in. Does it makes sense to also reach out to the team to check if it is a bug they are aware of.

Reply:
I agree I hated overriding some default values, but hated even more the -2px.

After investigating more by playing with the elastic-charts repo, the problem is actually with the use of the `EuiTooltip`. Removing it fixes the alignment issue. I [posted on their channel](https://elastic.slack.com/archives/CFK9BJ248/p1664563298017939) , let's see it I'm doing something wrong.
For now I decided to remove the `EuiToolTip`, it will take 30 seconds to add it back once we figure out how to do it properly...

Reply:
well actually with the help of the charts team I think we have a better `hack` for now. We're using `style={{ height: '100%' }}` on the EuiButtonIcon. It's either that or we don't use EuButtonIcon and have to style everything ourselves, like they do in the elastic-charts Storybook [here](https://github.com/elastic/elastic-charts/blob/master/storybook/stories/utils/components/get_legend_action.tsx#L81).

The charts team isn't really testing their components with Eui components so it's not the first time they're seeing those weird UI issues...

Reply:
created a [bug ticket](https://github.com/elastic/elastic-charts/issues/1832) on the elastic-charts repo for traceability